rpc/doc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Overview
+// Package rpc provides the underpinnings of requests and responses (server.Request)
+// and dispatching them to the API by defining a request
+// rpc provides the idea of a request that has a named type, a version, and a method to be called
+// and provides a mechanism – Find method – for matching a request to a callable function.
+
+// Overall Sequence Flow
```

Comment:
Here and everywhere else: Please leave an empty line after the title and don't use title case.

Reply:
updated